EXPhotoViewer/EXPhotoViewer.m

```diff
@@ -58,6 +58,7 @@ -(void)loadView {
     UIImageView* imageView = [[UIImageView alloc] initWithFrame:self.view.bounds];
     imageView.contentMode = UIViewContentModeScaleAspectFill;
```

Comment:
No prob, and thank you! Are you using cocoapods? I'll update the pod spec later but we'll either have to change the name of the pod or get the original repo owner to merge in the changes. Until then you'll need a line like:

```
pod 'EXPhotoViewer',        :git => 'https://github.com/ShowMobile/EXPhotoViewer.git', :commit => '807e807441fea0bc24c957b5a3e3c171c46e82ce'
```